src/main/java/org/jenkinsci/plugins/envinject/service/EnvInjectMasterEnvVarsRetriever.java

```diff
@@ -0,0 +1,14 @@
+package org.jenkinsci.plugins.envinject.service;
+
+import hudson.EnvVars;
+import jenkins.security.MasterToSlaveCallable;
+
+import java.io.IOException;
+import java.util.Map;
+
+public class EnvInjectMasterEnvVarsRetriever extends MasterToSlaveCallable<Map<String, String>, IOException> {
```

Comment:
Add `@Restricted(NoExternalUse.class)`?

Reply:
I already thought about this. But as most other classes in this package didn't use this annotation I did not add it for consistency reasons.

I'll just added it now.